### ISSUE TITLE
What kind of accuracy logic is used in neural_compressor.experimental.Benchmark?

### ISSUE THREAD

**Issue by tarekradi** [intel/neural-compressor#296]
The Reference Kit here 

https://github.com/oneapi-src/visual-quality-inspection/blob/main/src/intel_neural_compressor/neural_compressor_inference.py 

does the following import..

from neural_compressor.experimental import Benchmark

then it runs the following code...

    evaluator = Benchmark(config_path)
    evaluator.model = int8_model
    # create benchmark dataloader like examples/tensorflow/qat/benchmark.py
    evaluator.b_dataloader = test_loader
    evaluator('accuracy')

    print("*"*50)
    print("Evaluating the FP32 Model")
    print("*"*50)
    evaluator = Benchmark(config_path)
    evaluator.model = model
    # create benchmark dataloader like examples/tensorflow/qat/benchmark.py
    evaluator.b_dataloader = test_loader
    evaluator('accuracy')

What kind of accuracy logic is being run behind the scenes?

**Comment by chensuyue** [comment]
Do you ask for the default accuracy metric?  
If there is no customized setting of benchmark accuracy metric, it will be `'topk': 1`
https://github.com/intel/neural-compressor/blob/6cb64f3d3a11b18db8a32c7ab0522f233445ae9c/neural_compressor/experimental/benchmark.py#L332  
https://github.com/intel/neural-compressor/blob/6cb64f3d3a11b18db8a32c7ab0522f233445ae9c/neural_compressor/conf/config.py#L1312

**Comment by tarekradi** [comment]
Thank you. So in other words, the default accuracy metric is "Top 1", right?

**Comment by chensuyue** [comment]
Yes, you are right.

**Comment by tarekradi** [comment]
What other accuracy models can i chose from? Sorry for the basic question.

**Comment by mengniwang95** [comment]
https://github.com/intel/neural-compressor/blob/master/docs/source/metric.md Hi @tarekradi, please refer to this link for other metrics.

**Comment by tarekradi** [comment]
Thanks
